documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -1,69 +1,255 @@
 
-<a name="createproj"></a>
-# Creating Your Project
+Please refer to the links above to download and install the necessary requirements.
+
+### Creating Your Project
 
 1) Launch Visual Studio
 2) Click "Create new project"  
  ![VS2019_Create_New_Project.PNG](./.attachments/VS2019_CreateProject_Markup.png)
-3) Select .NET Standard on the left, and choose "Class Library (.NET Standard)." Make sure that you are using .NET Standard 2.0 
+3) Select .NET Standard on the left, and choose "Class Library (.NET Standard)." Make sure that you are using .NET Standard 2.0, or a .NET version that supports .NET Standard 2.0 (such as .NET Core 3.1, .NET 6, etc.).
  ![VS2017_New_DotNetStandard_20_Project.PNG](./.attachments/VS2019_CreateProject_ClassLibrary_Markup.png)
 4) Name your project
 5) Click "Create"
 
-<a name="configure"></a>
-# Configuring Your Project
+### Configuring Your Project
 
 You should now have a solution with one project file.
 
-## Add the Microsoft.Performance.SDK NuGet Package
+#### Add the Microsoft.Performance.SDK NuGet Package
 
 [This documentation](https://docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio) describes how to add a NuGet package to a Visual Studio project. Following these instructions, add the `Microsoft.Performance.SDK` package from [nuget.org](nuget.org) to your project.
 
-### Picking your SDK version
+#### Picking your SDK version
 The version of the SDK you add to your project will determine which versions of SDK drivers your plugin will work with. For example, a plugin that depends on SDK version 0.109.2 will not work a version of an SDK driver that uses SDK version 1.0.0.
 
 To decide which version of the SDK to use, refer to the [known SDK driver compatability lists](../Known-SDK-Driver-Compatibility/Overview.md).
 
-## Install WPA for Debugging
+#### Install WPA for Debugging
 
 One way to debug an SDK plugin project is to use WPA. Before we setup our project for this, WPA will need to be installed. 
 Please see [Using the SDK/Installing WPA](./Installing-WPA.md) for more information how to install WPA.
 
-## Setup for Debugging Using WPA
+#### Setup for Debugging Using WPA
 
 1) Right click your project and select "Properties"
 2) Select the "Debug" tab on the left
    * In newer versions of Visual Studio, you may need to click "Open debug launch profiles UI"
 3) For "Launch", select "Executable"
 4) For the "Executable", place the path to the `wpa.exe` that you previously installed as part of the WPT
    * Typically this might be: `C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\wpa.exe`
-5) For "Command line Arguments", add `-addsearchdir <bin folder for your plugin>` (e.g. `-addsearchdir C:\MyAddIn\bin\Debug\netstandard2.1`)
+5) For "Command line Arguments", add `-addsearchdir <bin folder for your plugin>` (e.g. `-nodefault -addsearchdir C:\MyAddIn\bin\Debug\netstandard2.1`)
```

Comment:
I'm torn on this one.

On one hand, if someone is debugging their plugin, it would be nice to not have cross-talk and chatter in the diagnostic console from other plugins. 

On the other hand, if their plugin uses data from other included plugins' cookers, they need to load them too.

Reply:
If we do include `nodefault` we should explain why and what that flag means. Or we can say look at the help menu and use whatever commands you'd like, you probably need `addsearchdir`

Reply:
I added

> :information_source: If you're developing a plugin that uses `DataCookers` from other plugins, you may wish to **not** include the `-nodefault` flag.